CMakeLists.txt

```diff
@@ -0,0 +1,189 @@
+# The MIT License (MIT)
+#
+# Copyright (c) 2017 Aaron Bishop, Tommy Nguyen
+#
+# Permission is hereby granted, free of charge, to any person obtaining a copy
+# of this software and associated documentation files (the "Software"), to deal
+# in the Software without restriction, including without limitation the rights
+# to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+# copies of the Software, and to permit persons to whom the Software is
+# furnished to do so, subject to the following conditions:
+#
+# The above copyright notice and this permission notice shall be included in all
+# copies or substantial portions of the Software.
+#
+# THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+# IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+# FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+# AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+# LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+# OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+# SOFTWARE.
+
+cmake_minimum_required(VERSION 3.2 FATAL_ERROR)
+project(date VERSION 2.1 LANGUAGES CXX)
+
+set(_CMAKE_TOOLCHAIN_PREFIX llvm-)
+
+if(${CMAKE_SOURCE_DIR} STREQUAL ${CMAKE_CURRENT_SOURCE_DIR})
+    set(NEED_PARENT Off)
+    set(DATE_DEFAULT_ENABLE_TESTS On)
+else()
+    set(NEED_PARENT On)
+    set(DATE_DEFAULT_ENABLE_TESTS Off)
+endif()
+
+find_package(CURL)
+
+if(WIN32)
+    find_program(DATE_TZ_SHELL_REMOVE_DIRECTORY "rd")
+    find_program(DATE_TZ_SHELL_REMOVE_FILE "del")
+    find_program(DATE_TZ_SHELL_MOVE_FILE "move")
```

Comment:
What about `${CMAKE_COMMAND} -E rename ...`. In general: are [cmake file operations](https://cmake.org/cmake/help/latest/command/file.html) inappropriate here?